registration/src/main/proto/lobby/registration/conference.proto

```diff
@@ -70,10 +74,12 @@ message ConferenceAccessCode {
     string value = 1;
```

Comment:
Microsoft's code has 'Slug' in their API. Is ConferenceAccessCode is the thing, which is Slug in their code?